document_ftp/README.rst

```diff
@@ -0,0 +1,6 @@
+This is a support FTP Interface with document management system.
+================================================================
+
+With this module you would not only be able to access documents through OpenERP
+but you would also be able to connect with them through the file system using the
+FTP client.
```

Comment:
Please use the OCA template for README - particularly the contributor part is important here, add the initial authors in it + yourself